src/coreComponents/dataRepository/BufferOps_inline.hpp

```diff
@@ -1088,7 +1088,7 @@ Pack( buffer_unit_type * & buffer,
     else
     {
       sizeOfPackedChars += Pack< DO_PACKING >( buffer,
-                                               localIndex( -1 ) );
+                                               globalIndex( -1 ) );
```

Comment:
These were getting packed as a `localIndex`, but read as a `globalIndex`, which doesn't work if they are not the same underlying type.